src/ray/rpc/grpc_server.cc

```diff
@@ -48,7 +47,13 @@ void GrpcServer::PollEventsFromCompletionQueue() {
         // incoming request.
         server_call->GetFactory().CreateCall();
         server_call->SetState(ServerCallState::PROCESSING);
-        main_service_.post([server_call] { server_call->HandleRequest(); });
+        if (!main_service_.stopped()) {
+          main_service_.post([server_call] { server_call->HandleRequest(); });
+        } else {
+          // TODO: should send reply here
```

Comment:
Can you finish this TODO in this PR? The change should be simple. Just call `SendReply` with an error status. 
(maybe we should rename `SendReply` to `Finish`)

Reply:
ok, i will chang it to Finish and invoke here